src/api/types.ts

```diff
@@ -1,121 +1,67 @@
 import { Stream } from '../util/grpc'
+import * as EventType from './typedef/event'
+import * as ExecutionType from './typedef/execution'
+import * as InstanceType from './typedef/instance'
+import * as ServiceType from './typedef/service'
 
-export type JSONString = string
 export type hash = string
 
-export type Event = {
-  hash?: hash
-  instanceHash: string
-  key: string
-  data: string
+export const ExecutionStatus = {
+  CREATED: 0,
```

Comment:
Are you sure about CREATED = 0? it was 1 before.

Reply:
```suggestion
  UNKNOWN: 0,
  CREATED: 1,
```

Reply:
also, why changing `enum` to `const`?

Reply:
And this enum is already defined in ` ExecutionType.types.Status`. You can just re-export it.